controls/pcidss_4.yml

```diff
@@ -2045,1274 +2629,1420 @@ controls:
+        - ensure_logrotate_activated
+
+  - id: '10.6'
+    title: Time-synchronization mechanisms support consistent time settings across all systems.
     levels:
       - base
     status: automated
     controls:
     - id: 10.6.1
-      title: 'System clocks and time are synchronized using time-synchronization
-        technology.Common time is established across all systems.Keeping
-        time-synchronization technology current includes managing vulnerabilities and
-        patching the technology according to PCI DSS Requirements 6.3.1 and 6.3.3.'
+      title: System clocks and time are synchronized using time-synchronization technology.
+      description: |-
+        Common time is established across all systems. Keeping time-synchronization technology
+        current includes managing vulnerabilities and patching the technology according to PCI DSS
+        Requirements 6.3.1 and 6.3.3.
       levels:
         - base
       status: automated
+      notes: |-
+        Maybe it is possible to optmize some similar rules related to ntp.
       rules:
-      - service_chronyd_or_ntpd_enabled
-      - ntpd_specify_remote_server
-      - package_chrony_installed
-      - service_ntp_enabled
-      - service_ntpd_enabled
-      - service_timesyncd_enabled
-      - chronyd_configure_pool_and_server      
+        - package_chrony_installed
+        - service_chronyd_or_ntpd_enabled
+        - service_ntp_enabled
+        - service_ntpd_enabled
+        - service_timesyncd_enabled
 
     - id: 10.6.2
-      title: 'Systems are configured to the correct and consistent time'
+      title: Systems are configured to the correct and consistent time.
+      description: |-
+        - One or more designated time servers are in use.
+        - Only the designated central time server(s) receives time from external sources.
+        - Time received from external sources is based on International Atomic Time or Coordinated
+          Universal Time (UTC).
+        - The designated time server(s) accept time updates only from specific industry-accepted
+          external sources.
+        - Where there is more than one designated time server, the time servers peer with one
+          another to keep accurate time.
+        - Internal systems receive time information only from designated central time server(s).
       levels:
         - base
       status: automated
+      notes: |-
+        The selected rules might need updates in order to restrict their applicabilities or their
+        prodtypes to avoid conflicts.
       rules:
-      - var_multiple_time_servers=suse
-      - ntpd_specify_remote_server
-      - ntpd_specify_multiple_servers
-      - chronyd_specify_remote_server
-      - chronyd_or_ntpd_specify_multiple_servers
+        # specific products should update the variable value when defining the profile based on
+        # this control file.
+        - var_multiple_time_servers=generic
+        - chronyd_specify_remote_server
```

Comment:
It can the adjusted in profile level by excluding eventual conflicting rules for a product. But in this initial version I agree to adopt a more conservative approach and move it to related_rules.
Done in cbf4fab4eb97774d042f1feb38633975c75f8506